tests/pytype_test.py

```diff
@@ -97,13 +141,16 @@ def pytype_test(args):
         runs += 1
 
         if code:
-            print("pytd error processing \"%s\":" % test_run.args[0])
             print(stderr)
             errors += 1
 
-    print("Ran pytype with %d pyis, got %d errors." % (runs, errors))
+    print('Ran pytype with %d pyis, got %d errors.' % (runs, errors))
     return max_code, runs
 
+def _get_module_name(filename):
```

Comment:
Could you move this above the location where it's called?

Reply:
Done